control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -683,7 +685,7 @@ func finalizerSecret(object metav1.Object) string {
 }
 
 func (r *Reconciler) getCaCerts() (string, error) {
-	secret, err := r.SecretLister.Secrets(r.SystemNamespace).Get(brokerIngressTLSSecretName)
+	secret, err := r.SecretLister.Secrets(eventingNamespace).Get(brokerIngressTLSSecretName)
```

Comment:
@pierDipi I had to change this because we are installing the secrets into the `knative-eventing` namespace currently (https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/data-plane/config/broker-tls/broker-ingress-tls-certificate.yaml#L19)

Reply:
I'd leave it to be SystemNamespace, if namespaced broker fails, we should disable TLS for namespaced broker for now until we discuss how (and if) do we want to solve TLS for namespaced broker